### DISCUSSION TITLE
关于软件收费问题！！！

### DISCUSSION THREAD

**Discussion by Hunlongyu** [Hunlongyu/ZY-Player#776] [公告]
软件一直都是免费使用的！但是有一些不良商家，无论是直接贩卖官方软件，还是二次开发编译软件贩卖都非常可恶，因为软件里留了微信收款码，但是由于没有在软件里直接说明软件是免费的，导致偶尔收到一些被骗的人发来的辱骂信息。虽然我不是很在意，但是收到的多了，就很烦。

最后申明，软件从始至终都是开源免费的，如遇收费，请立即给差评并申请退费，再拉黑举报。

软件里也将不再放置开发者的收款码了。

**Comment by xu8117** [comment]
会不会考虑重新起个项目或者拉个分支，开发一个收费版本
